wire-ios-utilities/Source/AtomicInteger.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+@objc
+public class AtomicInteger: NSObject {
```

Comment:
Since in ObjC it was actually common to have prefixes to types, I tend to keep the name `ZMAtomicInteger` for objc.
What do you think?
The filename is fine, since at some point we will just remove any `@objc` annotation.
```suggestion
@objc(ZMAtomicInteger) @objcMembers
public class AtomicInteger: NSObject {
```